configure.ac

```diff
@@ -27,11 +28,12 @@ PKG_CHECK_MODULES([SQLITE3], [sqlite3],
 PKG_CHECK_MODULES([ZMQ], [libzmq])
 
 AC_CONFIG_FILES([
-  Makefile \
-  xayagame/Makefile \
-  mover/Makefile \
-  mover/gametest/Makefile
+      Makefile \
+      xayagame/Makefile
+	  mover/Makefile \
+	  mover/gametest/Makefile
 ])
+
```

Comment:
The changes here seem to be purely reformatting / whitespace - please revert them.